R/set_args.R

```diff
@@ -26,7 +26,8 @@ set_window_args <- function(object, fun.name, ..., legend.name=NULL, side=c(1,2)
   else
     arguments = set_args(fun.name, package=package)
   e.fun = dots$e.fun
-  to.gsplot <- list(list(arguments = arguments, gs.config=list(legend.name = legend.name, side = side))) %>% 
+  to.gsplot <- list(list(arguments = append(formal_arguments(arguments, def.funs, names(config(fun.name))), window_arguments(arguments, def.funs)),
+                         gs.config=list(legend.name = legend.name, side = side, par=par_arguments(arguments, def.funs)))) %>% 
```

Comment:
in the call, need to do `def.funs=c('graphics::text','graphics::text.default')` I think (in text.R)